src/features/query-bar/constants.js

```diff
@@ -20,47 +19,35 @@ export const FLOAT: 'float' = 'float';
 export const ENUM: 'enum' = 'enum';
 export const DATE: 'date' = 'date';
 
+export const IS: '=' = '=';
+export const IS_NOT: '!=' = '!=';
+export const GREATER_THAN: '>' = '>';
+export const LESS_THAN: '<' = '<';
+
+export const IS_OPERATOR = {
+    displayText: 'is',
+    key: IS,
+};
+
+export const GREATER_THAN_OPERATOR = {
+    displayText: 'is greater than',
+    key: GREATER_THAN,
+};
+
+export const LESS_THAN_OPERATOR = {
+    displayText: 'is less than',
+    key: LESS_THAN,
+};
+
+export const IS_NOT_OPERATOR = {
+    displayText: 'is not',
+    key: IS_NOT,
+};
+
 export const COLUMN_OPERATORS = {
-    string: [
-        {
-            displayName: 'is',
-            type: STRING,
-        },
-    ],
-    number: [
-        {
-            displayName: 'is',
-            type: NUMBER,
-        },
-    ],
-    float: [
-        {
-            displayName: 'is',
-            type: FLOAT,
-        },
-    ],
-    enum: [
-        {
-            displayName: 'is',
-            type: ENUM,
-        },
-    ],
-    date: [
-        {
-            displayName: 'is',
-            type: DATE,
-        },
-        {
-            displayName: 'is greater than',
-            type: DATE,
-        },
-        {
-            displayName: 'is less than',
-            type: DATE,
-        },
-        {
-            displayName: 'is not',
-            type: DATE,
-        },
-    ],
+    string: [IS_OPERATOR, IS_NOT_OPERATOR, GREATER_THAN_OPERATOR, LESS_THAN_OPERATOR],
+    number: [IS_OPERATOR, IS_NOT_OPERATOR, GREATER_THAN_OPERATOR, LESS_THAN_OPERATOR],
+    float: [IS_OPERATOR, IS_NOT_OPERATOR, GREATER_THAN_OPERATOR, LESS_THAN_OPERATOR],
+    enum: [IS_OPERATOR, IS_NOT_OPERATOR, GREATER_THAN_OPERATOR, LESS_THAN_OPERATOR],
+    date: [IS_OPERATOR, IS_NOT_OPERATOR, GREATER_THAN_OPERATOR, LESS_THAN_OPERATOR],
```

Comment:
If each row is the same, then what's the purpose of having a separate list per type? Can we reduce to a single list?

Reply:
Yes I can have one list with all the operators and have all the types reference that list.